pkgs/applications/audio/csound/plugins.nix

```
@@ -0,0 +1,78 @@
+{ lib, stdenv, fetchFromGitHub, cmake, gettext, runCommand
+, csound
+, libjack2 ? null
```

Never use null as default. It can have nasty consequences.

If you need to control what to include or exclude, use guard flags, like

```suggestion
, enableLibjack2 ? true
, libjack2
```

---

What are the nasty consequences?

---

I have added `enableX` flags for each plugin.

---

This is hard to explain. A concrete example is better:

https://github.com/NixOS/nixpkgs/pull/233301#issuecomment-1567661913